CONTRIBUTING.md

```diff
@@ -8,49 +8,85 @@ This file is maintained at https://github.com/mozilla-iam/mozilla-iam/blob/maste
+All IAM repositories should follow these working agreements and include:
+
+1. A master branch which is tagged for releases.
+2. The settings documented in [GitHub-Security-Settings.md](GitHub-Security-Settings.md) are set for all repositories.
+3. Code contains some level of unit testing.
+4. Testing includes a linter or syntax analyzer such as `flake8`
+5. A README which indicates the project's purpose and a diagram of functionment.
+6. The code can always be tested locally without any credentials.
+7. Use `makefiles` with clear targets to build, run, deploy the code.
+8. No un-reviewed code may go to production.
+9. Code is developed on personal forks and merged back to the repositories under <https://github.com/mozilla-iam> or
+  <https://github.com/mozilla>.
+10. New design or large functionality changes must go through an
+  [RRA](https://infosec.mozilla.org/guidelines/risk/rapid_risk_assessment.html).
+11. Pentests should be run periodically for high risk projects.
+
+### Branches, tags, and pull-requests
+
+* The `master` branch is used for development, and tags are created to mark point in time releases. These releases may
+  be promoted to a production deploy.
+
+* Additional branches may be created (including a `production` branch, but these should serve as a way to work on a
+project with multiple people, not as personal branches. Use your own fork of the project for personal work or feature
+you work on alone. If this changes, you can always push the personal branch to the upstream repository).
+
+* Proof of concepts do not require review, unless they're promoted to production code.
 
 Life of a commit:
 ```
 
- Personal       Pull-Req     mozilla-iam      Pull-Req        mozilla-iam        Pull-Req        mozilla-iam
- Fork/Repos    +--------->   master branch  +------------->   staging branch   +-------------->  production branch
-                                                               (optional)
-                      ^                              ^                                ^
-                      |                              |                                |
-                      |                              |                                |
-                      + Review required   +-------------------------------------------+
-                        Except POC repos
+ Personal       Pull-Req     mozilla-iam      Tag
+ Fork/Repos    +--------->   master branch  +----------> version 1.0
+                      ^                              ^
+                      |                              |
+                      |                              |
+                      + Review required   +----------+
+                        Except POC or configuration repos
```

Comment:
nit: configuration changes that will be applied in production should be reviewed

Reply:
clarified it